package.json

```diff
@@ -1,51 +1,51 @@
 {
-	"name": "notifications",
-	"description": "This app provides a backend and frontend for the notification API available in Nextcloud.",
-	"version": "2.16.0",
-	"author": "Joas Schilling <coding@schilljs.com>",
-	"license": "agpl",
-	"private": true,
-	"scripts": {
-		"build": "NODE_ENV=production webpack --progress --config webpack.js",
-		"dev": "NODE_ENV=development webpack --progress --config webpack.js",
-		"watch": "NODE_ENV=development webpack --progress --watch --config webpack.js",
-		"lint": "eslint --ext .js,.vue src",
-		"lint:fix": "eslint --ext .js,.vue src --fix",
-		"stylelint": "stylelint css/*.css css/*.scss src/**/*.scss src/**/*.vue",
-		"stylelint:fix": "stylelint css/*.css css/*.scss src/**/*.scss src/**/*.vue --fix"
-	},
-	"dependencies": {
-		"@nextcloud/auth": "^2.1.0",
-		"@nextcloud/axios": "^2.3.0",
-		"@nextcloud/browser-storage": "^0.2.0",
-		"@nextcloud/capabilities": "^1.0.4",
-		"@nextcloud/dialogs": "^4.0.1",
-		"@nextcloud/event-bus": "^3.1.0",
-		"@nextcloud/initial-state": "^2.0.0",
-		"@nextcloud/moment": "^1.2.1",
-		"@nextcloud/notify_push": "^1.1.3",
-		"@nextcloud/router": "^2.1.2",
-		"@nextcloud/vue": "^7.11.6",
-		"@nextcloud/vue-richtext": "^2.0.4",
-		"howler": "^2.2.3",
-		"v-click-outside": "^3.2.0",
-		"vue": "^2.7.10",
-		"vue-material-design-icons": "^5.2.0"
-	},
-	"engines": {
-		"node": "^16.0.0",
-		"npm": "^7.0.0 || ^8.0.0"
-	},
-	"browserslist": [
-		"extends @nextcloud/browserslist-config"
-	],
-	"devDependencies": {
-		"@nextcloud/babel-config": "^1.0.0",
-		"@nextcloud/browserslist-config": "^2.3.0",
-		"@nextcloud/eslint-config": "^8.2.1",
-		"@nextcloud/stylelint-config": "^2.3.0",
-		"@nextcloud/webpack-vue-config": "^5.5.0",
-		"babel-loader-exclude-node-modules-except": "^1.2.1",
-		"vue-template-compiler": "^2.7.14"
-	}
+  "name": "notifications",
```

Comment:
https://github.com/nextcloud/server/blob/cfe81050f56550df864da83e6589377f93b127a4/.editorconfig#L8-L9

Can we keep tabs 🙈